Arrangement_on_surface_2/doc/Arrangement_on_surface_2/CGAL/Arr_point_location_result.h

```diff
@@ -1,25 +1,4 @@
 /*!
```

Comment:
Probably forgot to remove this line.
(see: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-21/logs_master/Arrangement_on_surface_2.log)